bin/node/runtime/src/lib.rs

```diff
@@ -415,10 +421,7 @@ impl pallet_session::Trait for Runtime {
 	type WeightInfo = weights::pallet_session::WeightInfo<Runtime>;
 }
 
-impl pallet_session::historical::Trait for Runtime {
-	type FullIdentification = pallet_staking::Exposure<AccountId, Balance>;
-	type FullIdentificationOf = pallet_staking::ExposureOf<Runtime>;
-}
+impl pallet_session::historical::Trait for Runtime {}
```

Comment:
Why do we still need this trait? Maybe let's auto-implement it in case `ValidatorIndentification` is implemented for the runtime?

Reply:
Yeah, we could revisit this once the other design is done. I tried to keep the minimal changes when the design is still under discussion.